DQM/SiStripMonitorClient/interface/SiStripDcsInfo.h

```diff
@@ -86,6 +86,7 @@ class SiStripDcsInfo: public edm::EDAnalyzer {
     MonitorElement* DcsFractionME;
     int TotalDetectors;
     std::vector<uint32_t> FaultyDetectors;
+    std::map<uint32_t,uint32_t> NLumiDetectorIsFaulty;
```

Comment:
Second arg is a count of lumi sections in a run, not events, so I just used a map. ~ a few thousand is the most LS in a run I've ever seen. First arg is number of modules. I'll switch to unordered_map for good practice, thank you.